policies/setting-up-alarms.md

```diff
@@ -0,0 +1,79 @@
+# AWS CloudWatch Alarm Setup Policy for Booky Services
+
+### Purpose
+This policy outlines the steps for configuring AWS CloudWatch Alarms to monitor booky services, specifically focusing on AWS Lambda functions and AWS X-Ray for distributed tracing. The goal is to ensure that performance, availability, and error conditions are effectively tracked, and appropriate alerts are triggered when issues arise.
+
+### Scope
+This applies to all developers, DevOps engineers, and technical personnel responsible for managing and monitoring the booky services deployed on AWS Lambda.
+
+---
+
+## 1. Metrics to Monitor
+
+We’ll monitor key Lambda and X-Ray metrics to ensure everything is running smoothly:
+
+1. **Lambda Function Error Rate**
```

Comment:
Mas okay bang may colon nalang din after the title dito? Para lang ma-separate kasi nare-render sila in one line kasama nung text sa ibaba 😅 

![image](https://github.com/user-attachments/assets/c466cbf7-7844-4b8d-a4ef-39dbf3b3b416)

Same with lines 20 and 25